cmd/antrea-agent/agent.go

```diff
@@ -82,6 +83,10 @@ var excludeNodePortDevices = []string{"antrea-egress0", "antrea-ingress0", "kube
 // run starts Antrea agent with the given options and waits for termination signal.
 func run(o *Options) error {
 	klog.Infof("Starting Antrea agent (version %s)", version.GetFullVersion())
+
+	// Windows platform doesn't support Egress feature yet.
+	egressEnabled := features.DefaultFeatureGate.Enabled(features.Egress) && !runtime.IsWindowsPlatform()
```

Comment:
`features.DefaultFeatureGate.Enabled(features.Egress)` will return `true`.  I checked unsupportedFeaturesOnWindows only checks features that are set explicitly. This is why NodePortLocal (which is also enabled by default) is in unsupportedFeaturesOnWindows but the process doesn't panic when running on Windows.
How should we handle this situation? Changing the default value for windows, or keeping it as is (only check when features are set explicitly)? Current code can pass the feature check but I think we don't want to run egressController for windows as it would be dummy.